.snyk

```diff
@@ -0,0 +1,26 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
Is this needed for the new snyk check?

Reply:
This is created when either a you do ```snyk wizard``` or when a vulnerability PR is auto opened by snyk. Basically just defines which vulnerabilities to ignore (for 30 days, if there is no fix for the time being) or patches that are applied at build time